game/views.py

```
@@ -605,6 +606,27 @@ def verify_otp(request):
+                        'game/welcome_email.txt', 
+                        {'username': user.username}
+                    )
+                    
+                    send_mail(
+                        subject='Welcome to Checkora 🎉',
+                        message=welcome_message,
+                        from_email=settings.DEFAULT_FROM_EMAIL,
+                        recipient_list=[user.email],
+                        fail_silently=True,
+                    )
+                except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)